### DISCUSSION TITLE
How do I know which user created certain token?

### DISCUSSION THREAD

**Discussion by vitalykarasik** [community/community#24637] [Apps, API and Webhooks]
<p>I have a token ( ghp_*****) and need to know which organization’s user created it.   How I can get this info - using CLI, API, …?</p>
<p>Thanks,<br>
Vitaly</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Make a request to the <code>/user</code> API endpoint, authenticated with the token. That’ll give you the user owning the token, and you can go on from there.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/rest/reference/users#get-the-authenticated-user">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/rest/reference/users#get-the-authenticated-user" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/5814919/181085770-cebce2c2-da71-4191-99a5-ea244adffdef.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/rest/reference/users#get-the-authenticated-user" target="_blank" rel="noopener">Users - GitHub Docs</a></h3>

  <p>The Users API allows to get public and private information about the authenticated user.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by vitalykarasik** [comment]
<p>Many thanks, it works! (and shame on me for so stupid question…)</p>
